api/cds.proto

```diff
@@ -131,14 +155,18 @@ message Cluster {
   oneof protocol_options {
     // [#not-implemented-hide:]
     TcpProtocolOptions tcp_protocol_options = 12;
+
+    // Additional options when handling HTTP1 requests.
     Http1ProtocolOptions http_protocol_options = 13;
+
     // Even if default HTTP2 protocol options are desired, this field must be
     // set so that Envoy will assume that the upstream supports HTTP/2 when
     // making new HTTP connection pool connections. Currently, Envoy only
     // supports prior knowledge for upstream connections. Even if TLS is used
     // with ALPN, http2 must be specified. As an aside this allows HTTP/2
```

Comment:
Not your change, but this should be reworded to "*http2_protocols_options* must be specified".